<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodeDialog</class>
 <widget class="QDialog" name="EncodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encode video</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QPlainTextEdit" name="feedbackTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Executable:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="executablePathEdit">
       <property name="text">
        <string>ffmpeg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="executableBrowseButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Arguments (newlines will be replaced with spaces):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="argumentsHelpButton">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="argumentsTextEdit">
     <property name="plainText">
      <string>-f rawvideo
-pix_fmt yuv422p10le
-s %wx%h
-r %fpsn/%fpsd
-i pipe:0
-c:v prores_ks
-y
%sd/%sn.mov</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="metricsEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="processingProgressBar">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>%v / %m</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="framesLabel">
       <property name="text">
        <string>Frames:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fromFrameSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="toLabel">
       <property name="text">
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="toFrameSpinBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="wholeVideoButton">
       <property name="text">
        <string>Whole video</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startStopEncodeButton">
       <property name="text">
        <string>Start</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
